packages/ember-metal/lib/is_empty.js

```diff
@@ -39,7 +40,7 @@ export default function isEmpty(obj) {
 
   let objectType = typeof obj;
 
-  if (objectType === 'object') {
+  if (objectType instanceof EmberMap) {
```

Comment:
Sorry for letting this slide. What's the call here? Will investigate CI failure.